content/webapp/views/pages/concepts/concept/concept.helpers.ts

```diff
@@ -42,3 +42,251 @@ export const getThemeSectionHeading = (
 
   return `${tabLabel} this ${conceptTypeLabel}`;
 };
+
+type ConceptSection = {
+  display: boolean;
+  label?: string;
+};
+
+type ConceptConfig = {
+  [key: string]: {
+    displayName?: string;
```

Comment:
I wonder if we want to make them all required. If we change them all to use 

```
{
  display: boolean
  label?: string
  }
  ```

then when looking at their declaration below, we don't have to remember which fields they could have that they don't, we know it was a conscious decision to not display it. I think verbosity is good here. What do you think?